DOCS-9519 Exchange Audit Log Events

wwarner-salesforce opened the pull request.

wwarner-salesforce committed:
DOCS-9519 Exchange Audit Log Events

sarathecoubian commented on modules/ROOT/pages/audit-logging.adoc:
```diff
@@ -1132,6 +1132,20 @@ properties : Environment name
+==== Asset Reviews
+
+==== File Upload
+
```
```suggestion

[%header,cols="25a,15a,15a,15a,15a,30a"]
|===
|User action
|Obj Type
|Object
|Parent
|Action
|Payload

|Upload file
|Exchange file
|File ID
|N/A
|Create
|subaction: none +
properties: file key

|Delete file
|Exchange file
|File ID
|n/a
|Delete
|subaction: none +
properties: file key

|Update file
|Exchange file
|File ID
|N/A
|Update
|subaction: none +
properties: file key
|===

```

sarathecoubian commented on modules/ROOT/pages/audit-logging.adoc:
```diff
@@ -1132,6 +1132,20 @@ properties : Environment name
+==== API Metadata
+
+==== Asset Reviews
+
```
```suggestion

[%header,cols="25a,15a,15a,15a,15a,30a"]
|===
|User action
|Obj Type
|Object
|Parent
|Action
|Payload

|Create a comment
|Asset portal review comment
|Comment ID
|Asset portal review
|Create
|subaction: none +
properties: comment

|Delete a comment
|Asset portal review comment
|Comment ID
|Asset portal review
|Delete
|subaction: none +
properties: comment

|Update a comment
|Asset portal review comment
|Comment ID
|Asset
|Update
|subaction: none +
properties: comment

|Create a review
|Asset portal review
|Review ID
|Asset
|Create
|subaction: none +
|properties: review object

|Delete a review
|Asset portal review
|Review ID
|Asset
|Delete
|subaction: none +
properties: review object

|Update a Comment
|Asset portal review
|Review ID
|Asset
|Update
|subaction: none +
properties: review object
|===

```

wwarner-salesforce committed:
Asset Manager

sarathecoubian commented on modules/ROOT/pages/audit-logging.adoc:
```diff
@@ -1132,6 +1132,142 @@ properties : Environment name
+==== Public Portals
+
+==== API Metadata
+
```
```suggestion

[%header,cols="25a,15a,15a,15a,15a,30a"]
|===
|User action
|Obj Type
|Object
|Parent
|Action
|Payload

|Create an API instance
|API instance
|API instance ID
|N/A
|Create
|subaction: none +
properties: name, isPublic

|Delete an API instance
|API instance
|API instance ID
|N/A
|Delete
|subaction: none +
properties: name, isPublic
|Update an API instance
|API instance
|API instance ID
|N/A
|Update
|subaction: none +
properties: name, isPublic
|===

```

wwarner-salesforce committed:
Asset Reviews

wwarner-salesforce committed:
File Upload

wwarner-salesforce committed:
API Metadata

wwarner-salesforce committed:
Asset Portals

sarathecoubian commented on modules/ROOT/pages/audit-logging.adoc:
```diff
@@ -1132,6 +1132,142 @@ properties : Environment name
+==== Asset Portals
+
+==== Public Portals
+
```
```suggestion

[%header,cols="25a,15a,15a,15a,15a,30a"]
|===
|User action
|Obj Type
|Object
|Parent
|Action
|Payload

|Update a domain
|Public portal domain
|Organization ID and domain 
|Public portal
|Update
|subaction: none +
properties: portal ID, domain

|Delete a domain
|Public portal domain
|Organization ID and domain 
|Public portal
|Delete
|subaction: none +
properties: portal ID, domain

|Create a page
|Public portal page
|Page path
|Public portal
|Create
|subaction: none +
properties: portal ID, page ID

|Delete a page
|Public portal page
|Page path
|Public portal
|Delete
|subaction: none +
properties: portal ID, page ID

|Update a page
|Public portal page
|Page path
|Public portal
|Update
|subaction: none +
properties: portal ID, page ID

|Create a portal
|Public portal
|Organization ID
|N/A
|Create
|subaction: none +
properties: project ID, created by ID

|Publish a portal
|Public portal
|Organization ID
|N/A
|Publish
|subaction: none +
properties: project ID, created by ID

|Delete a portal
|Public portal
|Organization ID
|N/A
|Delete
|subaction: none +
properties: project ID, created by ID

|Update a portal
|Public portal
|Organization ID
|N/A
|Update
|subaction: none +
properties: project ID, created by ID
|===

```

wwarner-salesforce committed:
Public Portals

wwarner-salesforce committed:
Update audit-logging.adoc

sarathecoubian commented on modules/ROOT/pages/audit-logging.adoc:
```diff
@@ -1132,6 +1132,400 @@ properties : Environment name
+|Action
+|Payload
+
+|Create an Asset
```
```suggestion
|Create an asset
```

wwarner-salesforce committed:
Update audit-logging.adoc

wwarner-salesforce committed:
Update audit-logging.adoc

wwarner-salesforce committed:
Update audit-logging.adoc

wwarner-salesforce committed:
Update audit-logging.adoc

wwarner-salesforce committed:
Mariano fixes

'asset manager' -> 'assets', 'api metadata' and 'file upload' after 'asset portals'
